blazor/datagrid/columns.md

```diff
@@ -511,6 +511,69 @@ In the following code sample, you can prevent default filter query generation us
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-prevent-query-generation-for-foriegnkey-column).
 
+### Enable multiple foreign key columns
+
+The Grid has an option to show multiple foreign key columns. This can be achieved by defining multiple `GridForeignColumn` directives in the Grid component.
```

Comment:
The Grid component has support to display a value in GridColumn which has foreign key behavior. Grid has support to show multiple foreign key columns in Grid. (i.e.) datasource from different tables with foreign key behavior can be displayed in Grid and perform Data /CRUD operations.